sql/drop.go

```diff
@@ -87,3 +88,67 @@ func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
 	}
 	return &valuesNode{}, nil
 }
+
+// DropDatabase drops a database.
+func (p *planner) DropDatabase(n *parser.DropDatabase) (planNode, error) {
+	if n.Name == "" {
+		return nil, errEmptyDatabaseName
+	}
+
```

Comment:
actually, not root. we should check that WRITE privilege on the database descriptor.
Could you also add a "privileges" comment similar to other statements?
```
privileges: WRITE on database
   notes: postgres requires the database owner
          mysql requires DROP privileges on the database
```